index.html

```diff
@@ -1,177 +1,241 @@
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="work-used-tools">
+          <li class="tool">html</li>
+          <li class="tool">css</li>
+          <li class="tool">javascript</li>
+        </ul>
+        <button class="portfolio-btn">See Project</button>
+      </div>
+  </section>
+  <section id="section-3" class="portfolio-section">
+    <div class="portfolio-summary">
+      <h1 class="title main-head">About Myself</h1>
+      <p class="description">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="#" class="blue-text" class="connect-link">LET’S CONNECT</a>
+      <ul class="contact-list">
+        <li>
+          <a href="#">
+            <img src="icons/twitter-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons/Linkedin-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons/m-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons/github-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons/Vector-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+      </ul>
+      <button class="portfolio-btn">Get my resume</button>
+    </div>
+    <ul class="skills">
+      <li class="skill">
+        <div class="skill-title">
+          <h2 class="domain-name">Languages</h2>
+          <img src="icons/down-arrow.png" class="more-icon" />
+        </div>
+        <ul class="languages-list">
+          <li class="language">
+            <img src="images/javascript-icon.png" class="language-icon" />
+            <h3 class="language-name">Javascript</h3>
+          </li>
+          <li class="language">
+            <img src="images/html-icon.png" class="language-icon" />
+            <h3 class="language-name">HTML</h3>
+          </li>
+          <li class="language">
+            <img src="images/css-icon.png" class="language-icon" />
+            <h3 class="language-name">CSS</h3>
+          </li>
         </ul>
-    </section>
+      </li>
+      <li class="unactive-skill">
+        <h3 class="domain-name">Frameworks</h3>
+        <img src="icons/right-arrow.png" class="more-icon" />
+      </li>
+      <li class="unactive-skill">
+        <h3 class="domain-name">Skills</h3>
+        <img src="icons/right-arrow.png" class="more-icon" />
+      </li>
+    </ul>
+  </section>
+  <footer class="portfolio-footer">
+    <form class="footer-form" action="#" method="post">
```

Comment:
- [ ] As suggested by the previous reviewer your form isn't working, kindly set your form action to your formspree link:

## EXAMPLE
`    <form class="footer-form" action="https://formspree.io/f/{form_id}" method="post">
`